mosaicml_examples/builders.py

```diff
@@ -25,7 +25,9 @@ def build_callback(name, kwargs):
             return SpeedMonitor(window_size=kwargs.get('window_size', 1))
         return SpeedMonitorMFU(window_size=kwargs.get('window_size', 1),
                                gpu_flops_available=kwargs.get(
-                                   'gpu_flops_available', None))
+                                   'gpu_flops_available', None),
+                               gpu_cost_per_hour=kwargs.get(
+                                   'gpu_cost_per_hour', None))
```

Comment:
Something like this to future proof? ;)

```suggestion
                               gpu_cost_per_hour=kwargs.get(
                                   'gpu_cost_per_hour', os.environ.get('DEVICE_PRICE_PER_HOUR', None)))
```

Reply:
I moved this to the calculator instead of getting it in the builder.